test/org/rust/lang/inspections/RustInspectionsTest.kt

```diff
@@ -4,13 +4,23 @@ import com.intellij.codeInspection.LocalInspectionTool
 import org.rust.lang.RustTestCase
 
 class RustInspectionsTest : RustTestCase() {
+    val NONEXISTENT_MODULE_DECLARATION_HINT = "Create module file"
+
     override fun getTestDataPath() = "testData/org/rust/lang/inspections/fixtures"
 
     private inline  fun <reified T: LocalInspectionTool>doTest() {
         myFixture.enableInspections(T::class.java)
         myFixture.testHighlighting(true, false, true, fileName)
+
+    }
+
+    private inline  fun <reified T: LocalInspectionTool>doTestWithIntention(hint: String) {
+        doTest<T>()
+        myFixture.findSingleIntention(hint)
```

Comment:
This tests that the intention exists, but does not check the result, you should `invoke` it and make sure that the file actually exists. 

Check that the test indeed fails if you create wrong file (for example, if you change extension to `.sr` here https://github.com/alexeykudinkin/intellij-rust/pull/147/files#diff-183f61c4c270fbbd5ded3b3810f32d9dR18).